rpcserver.go

```diff
@@ -6051,6 +6093,40 @@ func (r *rpcServer) FundingStateStep(ctx context.Context,
 			return nil, err
 		}
 
+	// If this is a message to register a new shim that should be funded
```

Comment:
Same as above, wouldn't expect the user to themselves need to register for a specific PSBT shim up front. But perhaps you have some new workflows in mind?